index.js

```diff
@@ -137,18 +148,28 @@ export default class LinterJSCS {
     };
   }
 
+  static getFilePath(path) {
+    const relative = atom.project.relativizePath(path);
+    return relative[1];
+  }
+
+  static getConfig(filePath) {
+    return configFile.load(false,
+      path.join(path.dirname(filePath), this.configPath));
```

Comment:
Yes, I agree. I wanted to restructure the whole file because it is hard to reason about, but there are no tests so there was no way to know if I was breaking anything.